### PR TITLE
Add a common-utils module

### PR DESCRIPTION
**Pull request by SPalominos**
Add a common-utils module containing basic extension : Script and String.

Adds to Script class the Logger methods : error(), info(), warn, debug().
Adds to String prefix(), postfix() and compileRegex() methods.

### CODE REVIEW

File: common-utils/pom.xml [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+            <artifactId>junit-jupiter-engine</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-simple</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.locationtech.jts</groupId>
+            <artifactId>jts-core</artifactId>
```
Comment (ebocher) [inline common-utils/pom.xml]: necessary ?

File: pom.xml [hunk 2]
```diff
@@ -14,7 +14,9 @@
         <version>0.0.1-SNAPSHOT</version>
         <packaging>pom</packaging>
         <modules>
+                <module>common-utils</module>
                 <module>geometry-utils</module>
+                <module>overpass-utils</module>
```
Comment (ebocher) [inline pom.xml]: seems not here

File: common-utils/pom.xml (hunk 1 above)
Comment (SPalominos) [reply common-utils/pom.xml]: nope, fixed.

File: pom.xml (hunk 2 above)
Comment (SPalominos) [reply pom.xml]: Fixed

**Review by ebocher — APPROVED** [review]

**Pull request merged**